### DISCUSSION TITLE
Add articles to readme

### DISCUSSION THREAD

**Discussion by sharadhirao** [argoproj/argo-cd#20208] [Q&A]
Hi team, 
I have written few articles on argocd:
1. [Argo CD — A GitOps Tool for Continuous Delivery] (https://medium.com/@raosharadhi11/argo-cd-a-gitops-tool-for-continuous-delivery-12788d274516)
1. [Argo CD — Resource Hooks (PreSync, PostSync, Sync, and SyncFail) and Sync Waves] (https://medium.com/@raosharadhi11/argo-cd-resource-hooks-presync-postsync-sync-and-syncfail-and-sync-waves-e753c6f02d85)
1. [Argo Notifications with Argo CD] (https://medium.com/@raosharadhi11/argo-notifications-with-argo-cd-d6ca0dc54cbc)
1. [ArgoCD with Vault using argocd-vault-plugin] (https://medium.com/@raosharadhi11/argocd-with-vault-using-argocd-vault-plugin-dccbc302f0c2)

Can we add this to the argocd repo if yes how?

**Comment by kostis-codefresh** [comment]
Please open a Pull Request with your changes.
